src/components/Dashboard.tsx

```diff
@@ -94,38 +85,89 @@ const TemplateComponent: React.FC<{ template: TemplateInList }> = ({ template })
   )
 }
 
-const FilterComponent: React.FC<{
+const PanelComponent: React.FC<{
   template: TemplateDetails
-  filter: Filter
-  setFiltersCount: Function
-}> = ({ template, filter, setFiltersCount: setLoadingFilters }) => {
+  filters: Filter[]
+}> = ({ template, filters }) => {
   const templateType = template.code
-  const { loading, applicationCount } = useListApplications({
-    type: templateType,
-    perPage: 1,
-    ...filter.query,
-  })
+  const [loadedFiltersCount, setLoadedFiltersCount] = useState(0)
+  const [totalMatchFilter, setTotalMatchFilter] = useState<{ [key: number]: number }>(
+    filters.reduce((totalPerFilter, filter) => ({ ...totalPerFilter, [filter.id]: 0 }), {})
+  )
+
+  const arrayFilters = filters.reduce((arrayFilters: { [key: string]: string }[], element) => {
+    if (typeof element.query === 'object') {
+      const queryObj = element.query as { [key: string]: string }
+      arrayFilters.push(queryObj)
+    }
+    return arrayFilters
+  }, [])
+
+  const queryMultiFilters = {
+    templateCode: { equalToInsensitive: templateType },
+    ...constructOrObjectFilters(arrayFilters),
+  }
+
+  const { loading, applications } = useListApplications(
+    {}, // Passing empty to use already construct GraphQL query on queryMultiFilters
```

Comment:
Suggested comment re-wording: `// Passing empty as graphQLFilterObject is already constructed` (using suggested variable name, so don't have to write variable name _and_ an explanation of what it is)